### PR TITLE
Add Portuguese, French and German

### PR DESCRIPTION
**Pull request by diogoscf**
Title says it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ It's the end!
 
 _Spoken languages:_
 
+- Portuguese
```
Comment (albertonl) [inline README.md:80]: This should be underneath Spanish on the list

**Review by albertonl — CHANGES_REQUESTED** [review]
Remember Spanish gud @diogoscf

File: README.md (hunk 1 above)
Comment (diogoscf) [reply README.md:80]: No

Comment (albertonl) [reply README.md:80]: I have le power

**Comment by albertonl** [comment]
Guess who won

**Comment by albertonl** [comment]
https://github.com/albertonl/babyshark/commit/31c146f603606689d9fd473eb0b8f1abd67027c0

**Comment by diogoscf** [comment]
smh

**Comment by diogoscf** [comment]
this is an abuse of power

**Comment by albertonl** [comment]
ikr

**Pull request merged**